types/luxon/index.d.ts

```diff
@@ -329,21 +334,21 @@ declare module 'luxon' {
                     seperator?: string;
                 }
             ): string;
-            toISO(options?: Object): string;
+            toISO(options?: ISOTimeOptions): string;
             toString(): string;
             union(other: Interval): Interval;
         }
 
-        type Settings = {
-            defaultLocale: string;
-            defaultNumberingSystem: string;
-            defaultOutputCalendar: string;
-            defaultZone: Zone;
-            defaultZoneName: string;
-            now: Function;
-            throwOnInvalid: boolean;
-            resetCache(): void;
-        };
+        class Settings {
+            static defaultLocale: string;
+            static defaultNumberingSystem: string;
+            static defaultOutputCalendar: string;
+            static defaultZone: Zone;
+            static defaultZoneName: string;
+            static throwOnInvalid: boolean;
+            static now(): number;
```

Comment:
now is a property, with a getter and a setter. See https://github.com/moment/luxon/blob/master/src/settings.js#L20. It should be defined as 

    static now: number;

Reply:
now is a `callback for returning the current timestamp`, so it should be a function. on default, now has defined in https://github.com/moment/luxon/blob/master/src/settings.js#L5
so it should be like `static now: () => number`
and i have a question. is this different with `static now(): number`?

Reply:
You're absolutely right, I misread the code. I'm not a specialist, but as I see it designed on the JS code, I think using

    static now: () => number

would be clearer. It show that now is just a readable/writable property of type function that is meant to be replaced by another one, and not a static method of the class.